_docs/node-operator-requirements-service.md

```diff
@@ -0,0 +1,104 @@
+- [OAK-hosted Rococo Chain Spec](https://github.com/OAK-Foundation/OAK-blockchain/blob/master/node/res/neumann-rococo-testnet.json)
+
+## Turing Network
+- [Turing Chain Spec](https://github.com/OAK-Foundation/OAK-blockchain/blob/master/node/res/turing.json)
+- [Kusama Chain Spec](https://github.com/paritytech/polkadot/blob/master/node/service/res/kusama.json)
+
+----------------------------------------------------------------------------------------------------------------------------------
+
+## OnFinality
+
+### Step 1: Setup your OnFinality account
+Create an [OnFinality account](https://app.onfinality.io/signup), and add a payment method to your [Accounts](https://app.onfinality.io/account).
+
+### Step 2: Dedicated Node selection
+Once you've confirmed your account, you are ready to setup a node. Starting with navigating to the Dedicated Nodes section. 
+
+![onfinality-setup-1](../assets/img/node-operator/onfinality-setup-1.png)
+
+Click on *Deploy New Node* Button, and this will direct you to a multi-step node infrastructure setup.
+
+### Step 3: Node Setup
+*Step 3.1: Search for the right network*
+If you'd like to setup Turing, for example, search for "Turing" on the top right search box. Then click on that network spec.
+
+![onfinality-setup-2](../assets/img/node-operator/onfinality-setup-2.png)
+
+*Step 3.2: Node Configuration*
+The public network spec provided by the OAK Team has the recommended specs above, and is up to date. 
+- For naming, please provide a name that you can identify, especially if you have multiple nodes. 
+- Image Version: double check that this is in fact the latest node version to deploy. You can run into issues if you choose an older image for the node. Additionally, do a cross-reference with our [Github release page](https://github.com/OAK-Foundation/OAK-blockchain/releases) as well as Discord.
+- Collator: Make sure you select "Collator" so that your node can produce blocks once on-boarded. If you accidentally choose archive or full node, you'll have to go through the setup process again, as it is not easy to convert from one type of node to the other.
+- Cloud provider & region: We recommend changing up deployment locations and cloud providers. However, certain configurations can allow for a faster node sync time with OnFinality's [Lighting Restore](https://documentation.onfinality.io/support/Lightning-Restore.1651703824.html) feature.
+- Node & storage size: We recommend sticking with the recommended size based on the public network spec.
+
+![onfinality-setup-3](../assets/img/node-operator/onfinality-setup-3.png)
+
+*Step 3.3: Launch Configuration*
+The parachain and relay chain arguments should be up to date. However, it's always prudent to double check and cross-reference the respective chain specs. See relevant network sections above.
+
+![onfinality-setup-4](../assets/img/node-operator/onfinality-setup-4.png)
```

Comment:
We can remove the bootnode params from here as all the new chainspecs have embedded bootnodes.

Reply:
Also, do we want folks running with the `unsafe-rpc-external` and `unsafe-ws-external` flags. I don't know the implications, but it sounds like it could be dangerous.

Reply:
ah, you're right, we should have them lock that down after they rotate the keys

Reply:
I will update once we have that cleaned up in our network spec